bin/run-travis-browser-tests.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+var RSVP  = require('rsvp');
+var spawn = require('child_process').spawn;
+var execSync = require('child_process').execSync;
+
+function run(command, _args) {
+  var args = _args || [];
+
+  return new RSVP.Promise(function(resolve, reject) {
+    console.log('Running: ' + command + ' ' + args.join(' '));
+
+    var child = spawn(command, args);
+
+    child.stdout.on('data', function(data) {
+      console.log(data.toString());
+    });
+
+    child.stderr.on('data', function(data) {
+      console.error(data.toString());
+    });
+
+    child.on('error', function(err) {
+      reject(err);
```

Comment:
we may need to force kill the child here: https://nodejs.org/api/child_process.html#child_process_child_kill_signal